relay-sampling/src/config.rs

```diff
@@ -255,6 +207,39 @@ pub enum DecayingFunction {
     Constant,
 }
 
+impl DecayingFunction {
```

Comment:
by the way, the time_range field on `SamplingRule` says the following:

>If a time range is closed on at least one end, the rule is considered a decaying rule.  
 
Yet, decaying function is only run if both ends are closed, is there a contradiction here?